bumble/gatt.py

```diff
@@ -99,12 +99,21 @@
 GATT_VOLUME_OFFSET_CONTROL_SERVICE          = UUID.from_16_bits(0x1845, 'Volume Offset Control')
 GATT_COORDINATED_SET_IDENTIFICATION_SERVICE = UUID.from_16_bits(0x1846, 'Coordinated Set Identification Service')
 GATT_DEVICE_TIME_SERVICE                    = UUID.from_16_bits(0x1847, 'Device Time')
-GATT_MEDIA_CONTROL_SERVICE                  = UUID.from_16_bits(0x1848, 'Media Control Service')
-GATT_GENERIC_MEDIA_CONTROL_SERVICE          = UUID.from_16_bits(0x1849, 'Generic Media Control Service')
+GATT_MEDIA_CONTROL_SERVICE                  = UUID.from_16_bits(0x1848, '[LE Audio] Media Control Service')
```

Comment:
I think we should keep all GATT UUID names without a `[Profile]` prefix, so that they are just as listed in the BT spec (also, it is possible, in theory, that the same characteristic could be used in multiple profiles, so tying them to a specific one isn't safe)